.stickler.yml

```diff
@@ -5,6 +5,9 @@ linters:
     extensions: 'php'
     fixer: true
 
+files:
+  ignore: ['tests/comparisons/*']
```

Comment:
@markstory Any idea why this could be ignored in stickler still?
I copied this from https://github.com/cakephp/migrations/blob/master/.stickler.yml#L4-L5

Reply:
Are you aiming to have these files *not* checked? That is what this config would do.

Reply:
Unfortunately, FriendsOfCake is using the OAuth version of the integration so we won't see any logs here. If the GitHubApp integration was used we'd get better logs and annotations instead of comments.

Reply:
I have done Stickler's GitHubApp integration now for FOC.

Reply:
Stickler respects `phpcs.xml` right? So it would be better to add the ignores there @dereuromark.

Reply:
@ADmad it does.